chroma_feedback/producer/circle/core.py

```diff
@@ -11,6 +12,7 @@ def init(program : ArgumentParser) -> None:
 
 	if not ARGS:
 		program.add_argument('--circle-host', default = 'https://circleci.com')
+		program.add_argument('--circle-only-mine', default = 'false')
```

Comment:
Use this instead:

program.add_argument('--circle-only-mine', action = 'store_true'))